src/univariates.jl

```diff
@@ -405,7 +405,8 @@ The natural logarithm of the difference between the cumulative density function
 function logdiffcdf(d::UnivariateDistribution, x::Real, y::Real)
     # Promote to ensure that we don't compute logcdf in low precision and then promote
     _x, _y = promote(x, y)
-    _x <= _y && throw(ArgumentError("requires x > y."))
+    _x == _y && return -Inf
```

Comment:
```suggestion
```
Perhaps just wait for `u + log1mexp(v - u)` to produce an `Inf` (because `iszero(v-u)` which will be promoted to the right type?